examples/base/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@raincatcher/base",
+  "version": "1.0.0",
+  "description": "NPM package template for the RainCatcher project",
+  "main": "index.js",
+  "author": "feedhenry-raincatcher@redhat.com",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "del lib && tsc",
+    "test": "nyc mocha",
+    "prepublish": "npm run build"
+  },
+  "nyc": {
+    "include": [
+      "src/**/*.ts"
+    ],
+    "extension": [
+      ".ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ],
+    "check-coverage": true,
+    "lines": 75,
+    "functions": 100,
+    "branches": 80
+  },
+  "devDependencies": {
+    "@types/mocha": "^2.2.41",
+    "del-cli": "^1.0.0",
+    "mocha": "^3.4.2",
+    "nyc": "^11.0.1",
+    "source-map-support": "^0.4.15",
```

Comment:
The third one gives the best developer experience and speed, but it's a bit harder to achieve. Point 2 better reflects a prod environment of course, but we'll already run them like that on CI and on the acceptance tests.

For relaying information I was still going to add a section about this to the readme, but the base module would have the implementation. Any changes in the future would have to be replicated manually to the existing modules, but wouldn't be that hard to do and would be just a single PR instead of 20 because of the monorepo.

A separate setup to showcase unit tests with internal dependencies might be worth it, but it's not something that we should pollute the base module template with.